contracts/modules/GuestModule.sol

```diff
@@ -0,0 +1,115 @@
+pragma solidity ^0.6.8;
+pragma experimental ABIEncoderV2;
+
+import "../utils/SignatureValidator.sol";
+
+import "./commons/Implementation.sol";
+import "./commons/ModuleAuth.sol";
+import "./commons/ModuleHooks.sol";
+import "./commons/ModuleCalls.sol";
+import "./commons/ModuleUpdate.sol";
+import "./commons/ModuleCreator.sol";
+
+import "../interfaces/receivers/IERC1155Receiver.sol";
+import "../interfaces/receivers/IERC721Receiver.sol";
+
+import "../interfaces/IERC1271Wallet.sol";
+
+
+/**
+ * GuestModule implement an Arcadeum wallet without signatures, nonce or replay protection.
+ * executing transactions using this wallet is not an authenticated process, and can be done by any address.
+ *
+ * @notice This wallet should be considered for public usage, and should never container funds or be assigned to non-public roles.
```

Comment:
@Agusx1211 do you mean, "this wallet shouldn't" ?

Reply:
The GuestModule enforces no authentication, so any permissions given to an instance of the contract are just like giving the permissions to any Ethereum contract (because anyone can call the method `execute()`).